src/markdown_generators.js

```diff
@@ -329,27 +336,33 @@ const generateCourseSectionFrontMatter = (
   title,
   pageId,
   parentId,
+  isGrandChild,
   hasMedia,
   isMediaGallery,
   menuIndex,
+  listInLeftNav,
   courseId
 ) => {
   /**
     Generate the front matter metadata for a course section given a title and menu index
     */
   const courseSectionFrontMatter = {
     title:     title,
-    course_id: courseId,
-    menu:      {
+    course_id: courseId
+  }
+
+  if (!isGrandChild || (isGrandChild && listInLeftNav)) {
```

Comment:
I think this is equivalent to `!isGrandChild || listInLeftNav`, because whenever `isGrandChild` is false then we're doing

```js
!false || ...
```

and we don't even evaluate the expression after the `||`, but conversely when `isGrandChild` is `true` then we're basically doing

```js
!true || (true && listInLeftNav)
```

and `true && x` is always going to be `x`, so the expression `isGrandChild && listInLeftNav` will I think in all cases where it is evaluated be equal to the value of `listInLeftNav` anyhow.

Is that right?

Reply:
Yes, you're right.  Thank you for the detailed breakdown!